lib/components/result-view/result-view.js

```diff
@@ -159,14 +158,9 @@ class ResultViewComponent extends React.Component<Props> {
             overflowY: "auto"
           }}
         >
-          <Display
-            outputs={toJS(outputs)}
-            displayOrder={displayOrder}
-            transforms={transforms}
-            theme="light"
-            models={{}}
-            expanded
-          />
+          {toJS(outputs).map((output, index) => (
+            <Display output={output} key={index} />
```

Comment:
It may only need to be unique within each `map` function. I am not sure about this though. [Here are the relevant docs](https://reactjs.org/docs/lists-and-keys.html#keys) for more info.

Reply:
> [Keys Must Only Be Unique Among Siblings](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings)

My fault 😅 I misunderstood they should be globally unique.